### ISSUE TITLE
Media support

### ISSUE THREAD

**Issue by efernandesng** [ewnd9/anki-apkg-export#1]
Would be nice if we could add images and sounds files into deck cards.
If you support this feature, I can help to implement :-)

```javascript
const fs = require('fs');
const AnkiExport = require('anki-apkg-export').default;

const apkg = new AnkiExport('deck-name');

apkg.addMedia('picture.jpg', fs.readFileSync('/tmp/picture.jpg'));

apkg.addCard('card #1 front <img src="picture.jpg"/ >', 'card #1 back');
apkg.addCard('card #2 front', 'card #2 back');

const zip = apkg.save();

fs.writeFileSync('./output.apkg', zip, 'binary');
console.log(`Package has been generated: output.pkg`);
```

http://decks.wikia.com/wiki/Anki_APKG_format_documentation

**Comment by ewnd9** [comment]
Do you want to make a pull request?

**Comment by efernandesng** [comment]
I can do that :-) Maybe next week

**Comment by ewnd9** [comment]
Just published `v2.0.1` with breaking API, `save` is async now (due to `jszip` upgrade from 2.x to 3.x, needed to be able to generate archives with `Blob` files from fetch) and returns `Promise`.

```js
apkg
  .save()
  .then(zip => {
    fs.writeFileSync('./output.apkg', zip, 'binary');
    console.log(`Package has been generated: output.pkg`);
  })
  .catch(err => console.log(err.stack || err));
```
